eslint.rules.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    "@typescript-eslint/no-this-alias": "off",
+    "prefer-rest-params": "off",
+    "no-empty": "off", // for tests folder
+    "@typescript-eslint/no-empty-function": "off", // for tests folder
+    "@typescript-eslint/no-explicit-any": "off", // for tests folder
```

Comment:
Is there a way to only disable this rule within the test folder?

Reply:
yes, but it will require to maintain another eslint file in the "tests" folder. Is it ok with you ?

Reply:
yes this is fine

Reply:
I figured out how to override rules for specific files within the same eslint file